pkgs/applications/audio/virtual-ans/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, unzip
+, xorg
+, libGL
+, alsaLib
+, SDL2
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "virtual-ans-${version}";
+  version = "3.0.2c";
+
+  src = fetchurl {
+    url = "https://warmplace.ru/soft/ans/virtual_ans-${version}.zip";
+    sha256 = "1dd4d91yjmp7xfjcjlam8l55r4963ypjgll97n4gjvl72m1fq856";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Photoelectronic microtonal/spectral musical instrument";
+    homepage = "https://warmplace.ru/soft/ans/";
+    license = licenses.free;
+    platforms = platforms.linux;
```

This way it won't run on aarch64, which is currently causing ofborg to fail. (Also, later you add darwin support with the `startScript` thingy, shouldn't that be in here as well?)
```suggestion
    platforms = [ "x86_64-linux" "i686-linux" ]
```
In addition, `meta` is usually sorted to the bottom of the expression. I'm not quite sure if there's any strict rules on this, but it's probably better to keep it consistent.

---

Thanks, couldn't find how to do this. I was under the impression that it had to be something that can be found in `stdenv.platforms`.

How about Darwin? In theory it runs there too, though I don't have the means of testing it myself.

---

> How about Darwin? In theory it runs there too, though I don't have the means of testing it myself.

Alright, guess it's fine to keep it disabled than.